### DISCUSSION TITLE
my conky.config is not working, can you help me

### DISCUSSION THREAD

**Discussion by Hyphaed** [brndnmtthws/conky#2012] [Q&A]
my conky.config is not working, can you help me

error

```home/ferran/Dev/conky.configconky: desktop window (60000a) is subwindow of root window (37a)conky: window type - normalconky: drawing to created window (0x2c00001)conky: drawing to double buffersh: 10: Syntax error: Unterminated quoted stringsh: 10:```

code

```
conky.config = {
    update_interval = 1,
    double_buffer = true,
    alignment = 'top_right',
    gap_x = 10,
    gap_y = 10,
    use_xft = true,
    font = 'Inter-VF:size=10',
    own_window = true,
    own_window_type = 'normal',
    own_window_transparent = true,
    own_window_hints = 'undecorated,below,sticky,skip_taskbar,skip_pager',
};

conky.text = [[
${execi 1 bash -c '
core=1
for i in /sys/devices/system/cpu/cpu*/cpufreq/scaling_cur_freq; do
  freq=$(cat "$i")
  printf "CPU%d: %s MHz\n" "$core" "$freq"
  core=$((core + 1))
done
temp=$(sensors | grep "Package id 0:" | awk "{print \$4}")
echo ""
echo "CPU Temperature: $temp"
'}
]]
```

**Comment by Caellian** [comment]
try moving the script from `conky.text` to a separate script file and invoke it in a signle line, I'm pretty sure lua doesn't support multiline strings.
